### ISSUE TITLE
Search button/icon missing

### ISSUE THREAD

**Issue by securestep9** [OWASP/owasp.github.io#31]
Search button/icon is missing - it is needed to be there and clearly labelled - not just for a best practice web design, but also for compliance with accessibility standards for visually impaired users. It is also needed to be there for mobile users to click.
